libs/components/src/lib/slider/slider.spec.ts

```diff
@@ -139,6 +140,31 @@ describe('vwc-slider', () => {
 		});
 	});
 
+	describe('connotation', function () {
+		const possibleConnotations = [Connotation.Accent,
+			Connotation.CTA
+		];
+
+		it('should leave connotation class empty if not set', async function () {
+			possibleConnotations.forEach(connotation => {
+				expect(getControlElement(element)
+					?.classList
+					.contains(connotation))
+					.toEqual(false);
+			});
+		});
```

Comment:
Not sure if this test is really needed
```suggestion
		it('should not set any connotation classes when no connotation is set', async function () {
			possibleConnotations.forEach(connotation => {
				expect(getControlElement(element)
					?.classList
					.contains(`connotation-${connotation}`))
					.toEqual(false);
			});
		});
```